README.md

```diff
@@ -110,6 +118,9 @@ $ brew install cmake \
   leveldb \
   ossp-uuid \
   boost \ openssl
+
+# If building with GPS support
+$ brew install gps
```

Comment:
Its actually is just gps for OS X. Its the gps.h headers and such that is needed. However it might make sense to also install gpsd so I'll add that